packages/testlab/package.json

```diff
@@ -4,10 +4,12 @@
   "description": "A collection of test utilities we use to write LoopBack tests.",
   "main": "index",
   "scripts": {
-    "build": "tsc && npm run build:es2015",
-    "build:es2015": "tsc --target es2015 --outDir lib-es2015",
+    "build": "npm run build:es2015 && npm run build:es2017",
+    "build:es2015": "tsc --target es2015 --outDir lib6",
+    "build:es2017": "tsc --target es2017 --outDir lib",
```

Comment:
Might be smart to go with lib2015 for consistency, I'm OK either way (I just mapped it to lib6 for now).